apps/site/assets/ts/components/RouteCardHeader.tsx

```diff
@@ -29,19 +29,24 @@ const RouteCardHeader = ({
         </span>
         <div className="c-link-block__view-schedule">View schedule</div>
       </span>
-      {alerts.length > 0 && (
-        <a
-          className="c-link-block__inner-link"
-          href={`/schedules/${route.id}/alerts`}
-          title="alert"
-        >
-          {alertIcon("c-svg__icon-alerts-triangle m-tnm-sidebar__route-alert")}
-          <span>
-            {alerts.length}
-            {alerts.length > 1 ? " alerts" : " alert"}
-          </span>
-        </a>
-      )}
+      {console.log(alerts)}
+      {alerts !== undefined
+        ? alerts.length > 0 && (
```

Comment:
This could be simplified to just `{alerts?.length && (...}`. As is it looks like if `alerts` is null it will still try to run `alerts.length`